src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIBookingService.kt

```diff
@@ -70,21 +66,11 @@ class CommunityAPIBookingService(
 }
 
 data class AppointmentCreateRequestDTO(
-
-  @JsonFormat(pattern = "yyyy-MM-dd")
-  @NotNull val appointmentDate: LocalDate,
-
-  @JsonFormat(pattern = "HH:mm:ss")
-  @NotNull val appointmentStartTime: LocalTime,
-
-  @JsonFormat(pattern = "HH:mm:ss")
-  @NotNull val appointmentEndTime: LocalTime,
-
-  @NotNull val officeLocationCode: String,
-
-  @NotNull val notes: String,
-
-  @NotNull val context: String,
+  val appointmentStart: OffsetDateTime,
+  val appointmentEnd: OffsetDateTime,
+  val officeLocationCode: String,
+  val notes: String,
+  val context: String,
```

Comment:
is the removal of the not null annotation related to the date formatting changes?

Reply:
As a request object I didn't see what value they provided. Not null is enforced by Kotlin.